acceptance/setup/common/050_Setup_Broker.rb

```diff
@@ -2,19 +2,42 @@
 extend Puppet::Acceptance::InstallUtils
 require 'pxp-agent/test_helper'
 
-step 'Install build dependencies on master' do
-  MASTER_PACKAGES = {
+step 'Install build dependencies for broker' do
+  BROKER_DEP_PACKAGES = {
     :redhat => [
       'git',
       'java-1.8.0-openjdk-devel',
     ],
   }
-  install_packages_on(master, MASTER_PACKAGES, :check_if_exists => true)
+  install_packages_on(master, BROKER_DEP_PACKAGES, :check_if_exists => true)
 end
 
-step 'Clone pcp-broker to master' do
+NUM_BROKERS = 2
+have_broker_replica = NUM_BROKERS > 1
+
+step 'Clone pcp-broker to broker_hosts' do
   clone_git_repo_on(master, GIT_CLONE_FOLDER,
     extract_repo_info_from(build_git_url('pcp-broker', nil, nil, 'https')))
+
+  step 'Replace PCP test certs with the Puppet certs of this SUT' do
+    on(master, puppet('config print ssldir')) do |result|
+      puppet_ssldir = result.stdout.chomp
+      broker_ssldir = "#{GIT_CLONE_FOLDER}/pcp-broker/test-resources/ssl"
+      on master, "\\cp #{puppet_ssldir}/certs/#{master}.pem #{broker_ssldir}/certs/broker.example.com.pem"
```

Comment:
yeah, i left them in to be safe.  they are meant to remove any alias that exists on some machines.  beaker should probably do this for us...